custom_model_runner/datarobot_drum/drum/common.py

```diff
@@ -199,6 +202,8 @@ def config_logging():
                 "targetName": Str(),
                 Optional("positiveClassLabel"): Str(),
                 Optional("negativeClassLabel"): Str(),
+                Optional("classLabels"): Seq(Str()),
```

Comment:
Don't we want to combine multiclass labels and binary? At least on the args-parser level?
Because now we are going to provide either:
--target-type binary --positive-class-label --negative-class-label
or
--target-type multiclass --class-labels a b c

Maybe better to go with the later for both?

Reply:
as a counter, positive and negative class have meaning in the context of binary classification. I'd prefer to leave it as is and add support for using --class-labels to binary if asked.